src/Dev/BuildTask.php

```diff
@@ -62,9 +70,17 @@ abstract public function run($request);
     /**
      * @return bool
      */
-    public function isEnabled()
+    public function isEnabled(): bool
     {
-        return $this->enabled;
+        $isEnabled = $this->config()->get('enabled');
+        $isDefaultEnabled = Config::inst()->get(__CLASS__, 'default_enabled');
+
+        if ($isEnabled === null) {
+            Deprecation::notice('6.0', 'Null values not allowed in CMS6 ...');
```

Comment:
```suggestion
```
In hindsight we don't actually need the the Deprecation notice, because it 99% of cases it'll be the default `null` value, and we don't want to be telling people at upgrade time that they need to set `is_enabled = true` on all of their BuildTasks since it'll become the default in CMS 6